.ci/jobs/apm-integration-test-axis-pipeline-mbp-jjbb.yml

```diff
@@ -0,0 +1,46 @@
+---
+- job:
+    name: apm-integration-test-axis-pipeline-mbp-jjbb
+    display-name: APM Integration Test Downstream MBP
+    description: APM Integration Tests downstream job that runs an Agent version matrix
+      tests in a multibranch pipeline
+    project-type: multibranch
+    number-to-keep: '5'
+    days-to-keep: '1'
+    script-path: .ci/integrationTestDownstream.groovy
+    scm:
+    - github:
+        branch-discovery: all
+        discover-pr-forks-strategy: merge-current
+        discover-pr-forks-trust: permission
+        discover-pr-origin: merge-current
+        discover-tags: true
+        repo: apm-integration-testing
+        repo-owner: elastic
+        credentials-id: 2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken
+        ssh-checkout:
+          credentials: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+        build-strategies:
+        - tags:
+            ignore-tags-older-than: -1
+            ignore-tags-newer-than: -1
+        - regular-branches: true
+        - change-request:
```

Comment:
remove change request, we would not want to create downstream jobs per PR

Reply:
Updated